heartbeat/strategy.go

```diff
@@ -9,6 +9,9 @@ type Strategy interface {
 	// Touch ticks the item at `location:ID` in Redis.
 	Touch(location, ID string, pool *redis.Pool) (err error)
 
+	// Removes an item at `location:ID` in Redis.
+	Purge(location, ID string, pool *redis.Pool) (err error)
```

Comment:
We need a way to remove expired workers from the heartbeater; it's not functionality that was provided before. In the hashmap strategy, this involves running an HDEL, but other implementations of Strategy might do it differently...